sdk/monitor/azure-monitor-opentelemetry-exporter/README.md

```diff
@@ -169,13 +169,13 @@ contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additio
 [virtualenv]: https://virtualenv.pypa.io
 [ot_sdk_python]: https://github.com/open-telemetry/opentelemetry-python
 [application_insights_namespace]: https://docs.microsoft.com/azure/azure-monitor/app/app-insights-overview#how-do-i-use-application-insights
-[exporter_options]: https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/monitor/azure-opentelemetry-exporter-azuremonitor/azure/opentelemetry/exporter/azuremonitor/_options.py#L21
+[exporter_options]: https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_options.py#L21
```

Comment:
Should folder structure be /sdk/monitor/azure-monitor-opentelemetry-exporter/**azure-monitor**/opentelemetry/exporter since azure-monitor is the product and not azure, then monitor? Having azure and then monitor might imply that there might be other folders under `azure` besides `monitor`.

Reply:
from my understanding, that would change the namespaces which we agreed upon

Reply:
Yeah not a strong opinion here.